task_sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -48,9 +48,12 @@ class RuntimeTaskInstance(TaskInstance):
     model_config = ConfigDict(arbitrary_types_allowed=True)
 
     task: BaseOperator
+    _ti_context_from_server: TIRunContext | None = None
```

Comment:
Hide this from repr please

Reply:
https://docs.pydantic.dev/latest/concepts/fields/#validate-default-values

Reply:
@ashb Pydantic removes private fields from repr automatically:

```py
In [1]: from pydantic import BaseModel

In [2]: class A(BaseModel):
   ...:     a: int
   ...:     _b: int
   ...:

In [3]: A(a=1, _b=2)
Out[3]: A(a=1)

In [4]: repr(_3)
Out[4]: 'A(a=1)'
```

Reply:
Ohh TIL.